scripts/tag.sh

```diff
@@ -16,7 +16,10 @@
 
 set -euo pipefail
 
-main_branch="main"
+git_root="$(git rev-parse --show-toplevel)"
+main_branch="$(cat "$git_root/cardano-dev.yaml" | yq -o json | jq -r '."main-branch"')"
```

Comment:
This would require cardano-dev.yaml file. Can you link to an example from here, or provide it in the repo?

Reply:
```
cat ../cardano-api/cardano-dev.yaml
version:
main-branch: main
release-branch:
  prefix: release/
  suffix: -branch
```

Reply:
I'm going to convert this to a dotfile.

Reply:
By 'here' I meant the `tag.sh` file. 😃